docs/protocol/core/fees.mdx

```diff
@@ -1,16 +1,14 @@
 ---
-title: "Transfer Fees"
+title: "Dispatch Fees"
 description: "Understanding origin and interchain fees for Hyperlane cross-chain transfers"
 ---
 
 ## Fee Estimation
 
-When sending a cross-chain transfer with Hyperlane GMP, two fees are required:
+When sending a cross-chain message with Hyperlane GMP, two fees are paid:
 
-1. **Origin Fee** – gas to submit the transaction on the origin chain
-2. [**Interchain Fee**](/docs/protocol/core/interchain-gas-payment) – gas to deliver and execute the message on the destination chain, including the relayer fee
-
-Both are paid upfront on the origin chain to the specified IGP.
+1. **Origin Fee** – transaction fee on the origin chain (charged by the blockchain)
+2. [**Interchain Fee**](/docs/protocol/core/interchain-gas-payment) – charged to cover destination chain transaction fees
```

Comment:
would add that the relayer fees are here. it reads that this is just the gas fee on the destination chain

> charged to cover destination chain transaction fees and the cross-chain transfer fee, including the relayer fee

Reply:
do you mean to highlight the fact that there may be a spread applied on top of the destination tx fee?

Reply:
yeah exactly. so where are the relayer fees? (and i think this is also where you can specify a warp fee as well)